### DISCUSSION TITLE
Lynx: update the config flags to have --enable-externs

### DISCUSSION THREAD

**Discussion by poetaman** [Homebrew/discussions#3042] [Everyday usage]
Lynx is a text based web browser for terminal. When one comes across a link to a video there is no way it can display that, it provides a feature to open the links in external program (like system's browser). But by default the configuration does not enable that, one needs to pass configuration flag ``--enable-externs``. Its missing in home-brew's version of lynx, please add it.

Links: 
1) https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/lynx.rb
2) https://formulae.brew.sh/formula/lynx

**Comment by SMillerDev** [comment]
Please see https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request#formulae-related-pull-request

**Reply by poetaman** [reply]
Did just that :) thanks
